webpack/move_to_pf/react-bootstrap-select/index.js

```diff
@@ -52,8 +53,11 @@ class BootstrapSelect extends React.Component {
   render() {
     // TODO: these classes are required because foreman assumes that all selects should use select2 and jquery multiselect
     // TODO: see also http://projects.theforeman.org/issues/21952
+    const { noneSelectedText } = this.props;
+
     return <FormControl {...this.props}
-                        data-none-selected-text={__('Nothing selected')}
+                        mything={() => { doSomething }}
+                        data-none-selected-text={__(noneSelectedText)}
```

Comment:
I would remove the `__()` here since when this is moved to patternfly it won't be used anyway.